examples/src/main/java/com/flipkart/gjex/examples/helloworld/web/HelloWorldResourceConfig.java

```diff
@@ -31,10 +33,12 @@
 public class HelloWorldResourceConfig extends ResourceConfig {
 	
 	@Inject
-	public HelloWorldResourceConfig (HelloWorldResource1 helloWorldresource1, 
-			HelloWorldResource2 helloWorldresource2) {
+	public HelloWorldResourceConfig (HelloWorldResource1 helloWorldresource1,
+																	 HelloWorldResource2 helloWorldresource2,
+																	 RotationManagementResource rotationManagementResource) {
 		register(helloWorldresource1);
 		register(helloWorldresource2);
+		register(rotationManagementResource);
```

Comment:
This should not be required. We are registering rotationManagement as part of setting up jetty server

Reply:
Removed